packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_thread_action.tsx

```diff
@@ -110,7 +111,11 @@ export const TooltipWrapper = ({
   );
 };
 
-const getTooltipCopy = (action: ActionType) => {
+const getTooltipCopy = (action: ActionType, threadArchived: boolean) => {
+  if (threadArchived) {
+    return 'Thread is archived';
+  }
```

Comment:
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/1df028d7-486f-4bce-a281-28e397f2021f)

This does not seem to be working 🤔

Reply:
Fixed!